binarySearch/binarySearch.js

```diff
@@ -11,5 +11,19 @@
 **/
 
 const binarySearch = (nums, target) => {
-
+	let min = 0;
+	let max = nums.length - 1;
+	let mid = Math.floor(max/2);
+	const search = () => {
+		if (target === nums[mid]) {
+			return mid;
```

Comment:
very nice recursive solution here! Code is nice and neat, and even though you have wide tabs, they're consistent so it's easy to read!